components/Header.vue

```diff
@@ -68,42 +72,50 @@ onMounted(() => {
             :href="links.documentation"
             aria-label="documentation-link"
             class="header-link"
-          >Documentation</NuxtLink>
+          >
+            Documentation
+          </NuxtLink>
           <NuxtLink
             key="news"
             :class="{
-              'header-menu-link--active': path === '/news'
+              'header-menu-link--active': path === '/news',
             }"
             aria-label="news-link"
             class="header-link"
             href="/news"
-          >News</NuxtLink>
+          >
+            News
+          </NuxtLink>
           <NuxtLink
             key="proposals"
             :href="links.proposals"
             aria-label="proposals-link"
             class="header-link"
-          >Proposals</NuxtLink>
+          >
+            Proposals
+          </NuxtLink>
           <NuxtLink
             v-if="!featureFlags.minimal"
             key="assets"
             aria-label="assets-link"
             class="header-link"
             href="#"
-          >Assets</NuxtLink>
+          >
+            Assets
+          </NuxtLink>
         </div>
 
         <div
           :class="{
-            'header-buttons--minimal': featureFlags.minimal
+            'header-buttons--minimal': featureFlags.minimal,
           }"
           class="header-buttons"
         >
           <LinkButton :href="links.githubUrl" aria-label="contribute">
             Contribute
             <Icon name="code" />
           </LinkButton>
-          <LinkButton v-show="!featureFlags.minimal" aria-label="donate" href="#">
+          <LinkButton aria-label="donate" href="https://ko-fi.com/redotengine">
```

Comment:
I recommend putting this link into the links compostable so all links are in one place. I think there's a donate link in the footer too